Isabelle/Chi_Calculus/Communication.thy

```diff
@@ -77,11 +77,14 @@ lemma context_localization:
   shows "a \<triangleright>\<^sup>\<infinity> x. P x \<parallel> b \<triangleright>\<^sup>\<infinity> x. Q x \<approx>\<^sub>\<flat> a \<triangleright>\<^sup>\<infinity> x. P x \<parallel> b \<triangleright>\<^sup>\<infinity> x. (a \<triangleright>\<^sup>\<infinity> x. P x \<parallel> Q x)"
   sorry
 
+abbreviation distributor :: "[chan, chan list] \<Rightarrow> process" (infix "\<Rightarrow>" 100) where
```

Comment:
Indeed, we have to move `distributor` into `Communication` in order to base `loss` and `duplication` on it. What about moving also all other things currently in `Relaying` to `Communication`?

Reply:
Sounds good!